src/appsco-access-report-page.html

```diff
@@ -0,0 +1,258 @@
+<link rel="import" href="../bower_components/neon-animation/neon-animated-pages.html">
+<link rel="import" href="../bower_components/neon-animation/animations/fade-in-animation.html">
+<link rel="import" href="../bower_components/neon-animation/animations/fade-out-animation.html">
+<link rel="import" href="../bower_components/neon-animation/neon-animatable-behavior.html">
+<link rel="import" href="../bower_components/iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../bower_components/iron-media-query/iron-media-query.html">
+
+<link rel="import" href="components/page/appsco-content.html">
+<link rel="import" href="components/components/appsco-search.html">
+<!--<link rel="import" href="components/access-report/appsco-company-access-report.html">-->
+<!--<link rel="import" href="components/access-report/appsco-company-access-report-roles.html">-->
+<!--<link rel="import" href="components/access-report/appsco-company-access-report-resources.html">-->
+<!--<link rel="import" href="components/access-report/appsco-company-access-report-contacts.html">-->
+<!--<link rel="import" href="components/access-report/appsco-company-access-report-image.html">-->
+
+<dom-module id="appsco-access-report-page">
+    <template>
+
+        <style include="shared-styles webkit-scrollbar-style">
```

Comment:
Remove unnecessary CSS. This is empty page. There are no lists, icons or Info section on it.